pkg/apis/openebs.io/v1alpha1/cas_template_keys.go

```diff
@@ -31,39 +31,54 @@ package v1alpha1
 type TopLevelProperty string
 
 const (
-	// ConfigTLP is a top level property supported by cas volume
-	// engine
+	// ConfigTLP is a top level property supported by CAS template engine
 	//
-	// The policy specific properties are placed with
-	// ConfigTLP as the top level property
+	// The policy specific properties are placed with ConfigTLP as the
+	// top level property
 	ConfigTLP TopLevelProperty = "Config"
-	// VolumeTLP is a top level property supported by volume
-	// policy engine
+	// VolumeTLP is a top level property supported by CAS template engine
 	//
```

Comment:
Do we need to write TLP property for cstor pool in this file only? Or a separate file.
If in a separate file then file name should be e.g. cas_template_volume_keys , cas_template_cstorpool_key etc IMO.
What is your opinion?

Reply:
@sonasingh46 We can have TLP property of cstor pool in this file or can even define them in cstor operations file.
e.g. pkg/volume/volume.go is for volume operations
similarly pkg/cstor/cstor.go will be for cstor operations

Reply:
I am more inclined towards having the keys in this file only. This will ensure a single source for keys that will be used in the CAS framework.
Thanks for pointing it out.